repos/cli.proj

```diff
@@ -48,8 +48,8 @@
     <RepositoryReference Include="newtonsoft-json" />
     <RepositoryReference Include="nuget-client" />
     <RepositoryReference Include="roslyn" />
-    <RepositoryReference Include="sdk" />
-    <RepositoryReference Include="templating" />
+    <!--RepositoryReference Include="sdk" /-->
+    <!--RepositoryReference Include="templating" /-->
```

Comment:
I think templating should work now, I've got this + Davis' changes to enable cli-migrate and sdk building locally so I will update if I run into problems.